### PR TITLE
EOS-21906 Factory: Support for Service Config Keys

### PR DESCRIPTION
**Pull request by zahidsk**

### CODE REVIEW

**Review by nitin-seagate — CHANGES_REQUESTED** [review]
The change-set is huge? Please confirm

**Comment by zahidsk** [comment]
> The change-set is huge? Please confirm

I did just few line changes in below 2 files but still git is showing as whole file change, (I cross checked using line no)
cortx_setup/__init__.py
cortx_setup/api_spec.yaml

**Comment by zahidsk** [comment]
> > The change-set is huge? Please confirm
> 
> I did just few line changes in below 2 files but still git is showing as whole file change, (I cross checked using line no)
> cortx_setup/**init**.py
> cortx_setup/api_spec.yaml

identified and correction commited

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: lr-cli/cortx_setup/commands/config/set.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        self.logger.debug(f"Updating confstore key {key} with value {value}")
+
+        value = int(value) if value.isdigit() else value
+        Conf.set(
+            'node_info_index',
+            key,
+            value
+        )
+
+        Conf.save('node_info_index')
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/config/set.py:70]: Please add new line..

**Review by nitin-seagate — APPROVED** [review]

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Review by sradhanand-pati — APPROVED** [review]
Looks Good to me.
Please updatethe PR title.

**Review by mazinamdar — APPROVED** [review]
looks good , just make sure to add  Copyright data in lr-cli/cortx_setup/commands/config/__init__.py

**Comment by 83bhp** [comment]
The changes got landed via PR: https://github.com/Seagate/cortx-prvsnr/pull/5985

**Pull request closed without merging**